test/integration/element-test.js

```diff
@@ -24,22 +24,19 @@ suite('element methods', function() {
   });
 
   test('#findElement', function() {
-    var root = client.findElement('html');
-    assert.ok(root.findElement('body').displayed());
+    client.findElement('html');
```

Comment:
It throws if it can't find the element, so there's no need for an assertion unrelated to `findElement` here.